EIPS/eip-7848.md

```diff
@@ -0,0 +1,122 @@
+
+Note: Just because a upgrade was activated, this does not necessarily mean that the new software behaviors must generate block `VOTING_WINDOW_END` + 1 differently than the old software behaviors. Perhaps the new software behaviors will stipulate that only blocks at some later time will generate differently. Perhaps the new software behaviors include some other consensus change which does not change how blocks are created.
+
+## Rationale
+
+### Forking is no Longer Possible
+
+Since the merge, forking Ethereum Mainnet has become practically impossible. Validators stake valuable assets to participate in the network, so any rational validator will choose to upgrade only if they expect widespread adoption. If a validator expects 95% or more participants to upgrade, they should upgrade; if they expect only 5% or less, they should not. For intermediate cases, there is a threshold where a validator would rationally shut down (incurring a small penalty) rather than risk participating in the wrong network—which could result in slashing 100% of their staked ether (currently 16 Ether per share).
+
+Therefore, proper management of consensual upgrades is crucial.
+
+### Community Direction
+
+1. The Ethereum project and community do not have an official mission statement or vision. However, this proposal asserts that the Ethereum community would prefer Ethereum Mainnet to be a community-directed project.
+2. On-chain signaling of upgrades enables community direction in a way that is not possible today.
+
+### Decentralization
+
+1. Currently, upgrade notices are published on blog.ethereum.org.
+2. The ethereum.org website has opaque ownership; for example, domain ownership (via whois) is redacted for privacy, and the website footer does not list a legal publisher.
+3. An opaque organization unilaterally announcing upgrades to client software is antithetical to the concept of a community-directed project.
+
+Under this new upgrade process, it would appropriate for Ethereum Foundation to change the way it blogs about upgrades as follows:
+
+```diff
+- The Ethereum network will be undergoing a scheduled network upgrade.
++ Ethereum Foundation proposes that clients upgrade and asks you to act now.
+```
+
+### Window
+
+Using a voting window to count votes provides real-time on-chain feedback about upgrade readiness. The upgrade is activated only after the successful completion of the voting window.
+
+## Backwards Compatibility
+
+### Trademark
+
+The Ethereum Foundation (Stiftung Ethereum), Zug, Switzerland, owns the trademark “Ethereum.” As a result, if anybody publishes a proposed Ethereum Mainnet consensus client, the foundation may have the right to restrict marketing of that software as an “Ethereum” client. That also posess unique risks related to securities rules.
```

Comment:
```suggestion
```

Reply:
This language notes a specific risk to people implementing EIP-9174. It is relevant to implementers.

It is so relevant that it may in fact prevent anybody from ever implementing the EIP.

This note should be part of the EIP. But we are open to discuss further what the specific language should be. And that should be discussed after this is merged as draft status.